CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to this repository
+
+## Getting Started
+
+1. Always use the latest version of terraform.
+2. Try not to use unofficial modules. AWS official modules are recommended.
+
+## Folder Structure
+
+Create a folder name with a name as of the module
+
+<https://www.terraform.io/docs/language/modules/develop/structure.html>
+
+```text
+<module name>
+ ┣ modules
+ ┃ ┗ <sub-module>
+ ┃ ┃ ┣ main.tf
+ ┃ ┃ ┣ variables.tf
+ ┃ ┃ ┗ versions.tf
+ ┣ policies
+ ┃ ┣ iam.yml
+ ┃ ┣ rds.yml
+ ┃ ┣ route53_skip.yml
+ ┃ ┣ secretsmanager.yml
+ ┃ ┣ security_group.yml
+ ┃ ┗ security_group_skip.yml
+ ┣ tests
+ ┃ ┣ vars
+ ┃ ┃ ┣ test1.json
+ ┃ ┃ ┗ test2.json
+ ┃ ┣ main.tf
+ ┃ ┣ test.ps1
+ ┃ ┗ tests.ps1
+ ┣ main.tf
+ ┣ outputs.tf
+ ┣ README.md
+ ┣ variables.tf
+ ┗ versions.tf
+```
+
+- main.tf - contains terraform resources, data, modules.
+- version.tf - terraform required versions + providers
+- outputs.tf - All outputs of the module
+- variables.tf - Variables needed by the module
+- README.md - should contain Input Parameters and Example
+- policies - Policy Sentry Policies required by the module
+- tests - Tests that should be run on the module
+
+## Tests
+
+Tests should contain a test.ps1 which will say how to execute the tests. There should be a terraform main.tf which will contain the module that should be tested
```

Comment:
Is it alright to just copy-paste the test.ps1 in the rds module, or is there something else we should do?

Reply:
I removed the tests.ps1 and incorporated the logic in run.ps1. So no additional files will be required for the tests other than the tf files and vars files